testing/discovery_interface.py

```diff
@@ -27,6 +27,12 @@
 DEVELOPER_URL = "http://{}:{}/developer".format(DISCOVERY_HOST, DISCOVERY_PORT)
 
 
+def check_yaml(yaml_dir):
+    yamllint_output = subprocess.check_output('python3 -m yamllint {}; exit 0;'.format(yaml_dir), shell=True).decode('utf-8')
+    if 'error' in yamllint_output.lower():
+        raise Exception('\n\nInvalid yaml in {}, please fix the following errors:\n{}'.format(yaml_dir, yamllint_output))
```

Comment:
could do an `f""` string for fastness

Reply:
I didn't want to do an `f""` string because it isn't supported in python 3.5 which makes this less portable but I wish I could because it looks so much better than `"".format()` :laughing:

Reply:
ahhh this is the finos repo :+1: